### PR TITLE
Issue 3849: Slight changes to config_install_default_config().

### PR DESCRIPTION
**Pull request by ghost**
Addresses https://github.com/backdrop/backdrop-issues/issues/3849

### CODE REVIEW

File: core/includes/config.inc [hunk 1]
```diff
@@ -251,8 +251,7 @@ function config_load_multiple($names, $type = 'active') {
  */
 function config_install_default_config($module, $config_name = NULL) {
   $module_config_dir = backdrop_get_path('module', $module) . '/config';
-  $storage = new ConfigFileStorage($module_config_dir);
-  if (is_dir(backdrop_get_path('module', $module) . '/config')) {
+  if (is_dir($module_config_dir)) {
```
Comment (klonos) [inline core/includes/config.inc:254]: I see how I would do the same here @BWPanda 👍

**Review by drupol — APPROVED** [review]
:+1:

File: core/includes/config.inc [hunk 2]
```diff
@@ -262,6 +261,7 @@ function config_install_default_config($module, $config_name = NULL) {
       $file = array_pop($parts);
       $file_config_name = str_replace('.json', '', $file);
       if (is_null($config_name) || $file_config_name === $config_name) {
+        $storage = new ConfigFileStorage($module_config_dir);
```
Comment (klonos) [inline core/includes/config.inc]: Not sure what this does and how things work, so cannot comment on this change (other than making this comment here that is 😄)

Comment (drupol) [reply core/includes/config.inc]: I don't know either yet, but the proposed change make sense to me.

**Review by klonos — APPROVED** [review]
👍

File: core/includes/config.inc (hunk 2 above)
Comment (klonos) [reply core/includes/config.inc]: This seems to have been a mishap. @BWPanda fixed it. Everything makes sense now 😄

**Comment by quicksketch** [comment]
Merged into 1.x and 1.13.x. Thanks @BWPanda, @drupol, and @klonos!

**Pull request merged**